Sources/RingPublishingTracking/Public/Model/Audio/AudioEventParameter.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  AudioEventParameter.swift
+//  RingPublishingTracking
+//
+//  Created by Bernard Bijoch on 10/10/2024.
+//
+
+import Foundation
+
+/// Audio event parameter
+public struct AudioEventParameter: Encodable {
```

Comment:
to nie musi być publiczne? do prywatnego katalogu i bez public?

Reply:
Przeniesione